projects/Mallard/src/components/weather.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import { Text, StyleSheet, FlatList, View } from 'react-native'
+import { useCachedOrPromise } from 'src/hooks/use-cached-or-promise'
+import { withResponse } from 'src/helpers/response'
+import { Forecast } from '../common'
+import { metrics } from 'src/theme/spacing'
+import { WeatherIcon } from './weather/weatherIcon'
+import Moment from 'moment'
+import { fetchWeather } from 'src/helpers/fetch'
+
+const styles = StyleSheet.create({
+    weatherContainer: {
+        display: 'flex',
+        flexDirection: 'row',
+        width: 'auto',
+        height: 'auto',
```

Comment:
I'd precalculate this height since we always know how tall it's gonna be and then the page won't jump around after the weather loads

Reply:
Done.